### DISCUSSION TITLE
How to obtain the number of tokens for Prompt and Completion in streaming mode?

### DISCUSSION THREAD

**Discussion by Henry910** [waylaidwanderer/node-chatgpt-api#209] [Help] [answered]
Hello, in this project, if I set "stream: true", how can I obtain the token values for Prompt and Completion? Can you tell me how to get the value of token?

**Comment by waylaidwanderer** [comment, marked as answer]
OpenAI doesn't provide that information in streaming mode. You have to keep track of it yourself some other way.

**Reply by Henry910** [reply]
Yes, thank you very much.
